README.md

```diff
@@ -1,2 +1,232 @@
+    - [1- Html](#1--html)
+    - [2- CSS](#2--css)
+    - [3- Javascript](#3--javascript)
+    - [4- React](#4--react)
+    - [5- Redux](#5--redux)
+    - [6- Jest](#6--jest)
+    - [Tech Stack ](#tech-stack-)
+  - [🚀 Live Demo ](#-live-demo-)
+    - [Key Features ](#key-features-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [1- Web Browser](#1--web-browser)
+    - [2- Code Editor](#2--code-editor)
+    - [3- Git](#3--git)
+    - [4- Node.js](#4--nodejs)
+    - [5- NPM](#5--npm)
+    - [6- React](#6--react)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+    - [Deployment](#deployment)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Microverse React capstone project - Metrics webapp Module 3 <a name="about-project"></a>
+
 This React capstone project is about building a mobile web application to check a list of metrics (numeric values) that you will create making use of React and Redux.
+
+You will select an API that provides numeric data about a topic that you like and then build the webapp around it. The webapp will have several pages:
+
+one page with a list of items that could be filtered by some parameters; in the example below, it's a list of metrics that can be filtered by the country (imagine a search field to introduce the country name like Italy, Croatia, etc.). This page should be your homepage.
+one page for the item details; in the example, the detail page for Czech Republic cities with number of views. Use 'vercel --prod' to deploy the project to production. 
+
+The Weather App is a website designed to display weather conditions for cities along with their latitude and longitude coordinates. This website is primarily built using React and Redux. Users have the capability to search for specific locations to access their respective weather conditions. Additionally, the website is optimized for mobile devices, ensuring a responsive user experience. It also provides detailed information for each city's weather conditions.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### 1- Html
+### 2- CSS
+### 3- Javascript
+### 4- React
+### 5- Redux
+### 6- Jest
+
+
+
+### Tech Stack <a name="tech-stack"></a>
+
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a>Html</a></li>
+    <li><a>Css</a></li>
+    <li><a>Javascript</a></li>
+    <li><a>React</a></li>
+    <li><a>Redux</a></li>
+    <li><a>Jest</a></li>
+  </ul>
+</details>
+
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://www.loom.com/share/bf34dfe621194fcd8e2c0e9599a321ae?sid=9a9c451b-f469-4900-bb8f-099f419c19cb)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
```

Comment:
- [ ] As requested by the previous reviewer in the presentation video you should also highlight some interesting piece of code or something you built that you are very proud of. You should re-record the video again with all the below-mentioned steps.

![image](https://github.com/Abdulwhbhussain/Metrics-Webapp-Microverse/assets/83803414/598c3cf7-e01f-4bee-b555-986e745ad3da)